src/main/java/edu/ie3/datamodel/io/source/DataSource.java

```diff
@@ -5,11 +5,9 @@
 */
 package edu.ie3.datamodel.io.source;
 
-import edu.ie3.datamodel.io.connectors.DataConnector;
-
 /** Describes a class that fetches data from a persistence location */
 public interface DataSource {
 
   /** @return the connector of this source */
-  DataConnector getDataConnector();
+  //  DataConnector getDataConnector(); // todo check if we need this
```

Comment:
AFAIK @mia-krause is using this in here feature branch and it _might_ make sense to have it for database connections. I'll remove it anyway because adding it in later release is easier than removing it in the future ...